image-index.md

```diff
@@ -55,19 +55,29 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`os.version`** *string*
 
-        This OPTIONAL property specifies the operating system version, for example `10.0.10586`.
+        This OPTIONAL property specifies the operating system version.
```

Comment:
I'm unsure this property is a minimal requirement or an exact match.

cc @jhowardmsft

Does Windows already using this property?
If not, I think it is even ok to mark this property as "reserved, SHOULD NOT use".